src/main/java/seedu/multitasky/model/EntryBook.java

```diff
@@ -51,8 +70,28 @@ public EntryBook(ReadOnlyEntryBook toBeCopied) {
 
     //// list overwrite operations
 
-    public void setEntries(List<? extends ReadOnlyEntry> entries) {
-        this.entries.setEntries(entries);
+    public void setActiveList(List<? extends ReadOnlyEntry> entries) {
+        this._activeList.setEntries(entries);
+    }
+
+    public void setArchive(List<? extends ReadOnlyEntry> entries) {
+        this._archive.setEntries(entries);
+    }
+
+    public void setBin(List<? extends ReadOnlyEntry> entries) {
+        this._bin.setEntries(entries);
+    }
+
+    public void setEventList(List<? extends ReadOnlyEntry> entries) {
+        this._eventList.setEntries(entries);
+    }
+
+    public void setDeadlineList(List<? extends ReadOnlyEntry> entries) {
+        this._deadlineList.setEntries(entries);
+    }
+
+    public void setFloatingTaskList(List<? extends ReadOnlyEntry> entries) {
+        this._floatingTaskList.setEntries(entries);
     }
 
     public void setTags(Collection<Tag> tags) throws UniqueTagList.DuplicateTagException {
```

Comment:
Fixed, thanks. This is a code smell on my part, similar errors may be lurking elsewhere, lol.